### ISSUE TITLE
about building a blog

### ISSUE THREAD

**Issue by Repo-Kristx** [RupeshBhandari/RupeshBhandari.github.io#1]
It comes this way:
I tried to add a **theme chirpy** (same as yours) to my blog, but after i push the blog to github, I found my home page lost.
The **only** thing remain is:
`---layout: home# Index page---`
I tried so many time but no effect.
So I found your blog, which use this theme too.
I thought I can **fork** your code, run it. If it could run, I can change the things contains and turns it into my blog.
But I failed eventually... it says:
> The page build failed for the `main` branch with the following error:
> The jekyll-theme-chirpy theme is not currently supported on GitHub Pages. For more information, see https://docs.github.com/github/working-with-github-pages/adding-a-theme-to-your-github-pages-site-using-jekyll.

But we know that chirpy is ok to use, so I'm here try to get some answers...
**Did you ever had these situations since your building?**
If so, maybe we can talk more about it.
If not, maybe I should look for other ways to solve this...

Anyway, **thank you** for providing a complete blog page.

**Comment by RupeshBhandari** [comment]
Yeah there were few issues, we can talk about it in telegram. Id: rpshbhandari
